<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>820</width>
    <height>720</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>820</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Truckload Volume Calculator</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../assets/assets.qrc">
    <normaloff>:/icon/images/coontrol.png</normaloff>:/icon/images/coontrol.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>10</number>
    </property>
    <property name="leftMargin">
     <number>10</number>
    </property>
    <property name="topMargin">
     <number>10</number>
    </property>
    <property name="rightMargin">
     <number>10</number>
    </property>
    <property name="bottomMargin">
     <number>10</number>
    </property>
    <item>
     <widget class="QLabel" name="lbl_splash">
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap resource="../assets/assets.qrc">:/image/images/splash.png</pixmap>
      </property>
      <property name="scaledContents">
       <bool>false</bool>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="grb_scans">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="title">
       <string>Scans</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <property name="spacing">
        <number>10</number>
       </property>
       <property name="leftMargin">
        <number>10</number>
       </property>
       <property name="topMargin">
        <number>10</number>
       </property>
       <property name="rightMargin">
        <number>10</number>
       </property>
       <property name="bottomMargin">
        <number>10</number>
       </property>
       <item>
        <widget class="QTableWidget" name="tbw_scans">
         <property name="selectionMode">
          <enum>QAbstractItemView::SingleSelection</enum>
         </property>
         <property name="selectionBehavior">
          <enum>QAbstractItemView::SelectRows</enum>
         </property>
         <property name="sortingEnabled">
          <bool>false</bool>
         </property>
         <property name="cornerButtonEnabled">
          <bool>false</bool>
         </property>
         <column>
          <property name="text">
           <string>Date</string>
          </property>
          <property name="font">
           <font>
            <bold>true</bold>
           </font>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Volume</string>
          </property>
          <property name="font">
           <font>
            <bold>true</bold>
           </font>
          </property>
         </column>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="grb_actions">
      <property name="title">
       <string>Actions</string>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <widget class="QPushButton" name="btp_startScan">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="text">
          <string>Start Scan</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QPushButton" name="btp_stopScan">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Stop Scan</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QPushButton" name="btp_showPC">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="text">
          <string>Show Point Cloud</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QPushButton" name="btp_calculateVolume">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="text">
          <string>Calculate Volume</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="status_bar"/>
 </widget>
 <resources>
  <include location="../assets/assets.qrc"/>
 </resources>
 <connections/>
</ui>
